htdocs/compta/tva/quadri_detail.php

```diff
@@ -428,7 +427,10 @@
 		if (is_array($x_both[$rate]['coll']['detail'])) {
 			// VAT Rate
 			print "<tr>";
-			print '<td class="tax_rate">'.$langs->trans("Rate").': '.vatrate($rate).'%</td><td colspan="'.($span + 1).'"></td>';
+			print '<td class="tax_rate" colspan="' . ($span+1) . '">';
+			print $langs->trans('Rate') . ' : ' . vatrate($rate) . '%';
+			print ' - <a href="' . dol_buildpath('/compta/tva/quadri_detail.php', 1) . '?invoice_type=customer&amp;vat_rate_show=' . $rate . '&amp;year=' . $year_start . '&amp;month=' . $month_start . '">' . img_picto('', 'chevron-down', 'class="paddingrightonly"') . $langs->trans('VATReportShowByRateDetails') . '</a>';
```

Comment:
As this is an internal url, the dol_buildpath (designed for link to external url of modules) can be replaced with
DOL_URL_ROOT.'/compta/tva/quadri_detail.php'
Also any param into an url must be url encode, for example
&amp;vat_rate_show=' .urlencode($rate)